samoa-api/pom.xml

```diff
@@ -101,6 +101,16 @@
       <version>${hadoop.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>0.8.2.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka_2.10</artifactId>
+      <version>0.8.1</version>
```

Comment:
Why two separate versions for Kafka and Kafka-clients?
Also, is Kafka really needed?
And if so, should it be declared as provided?